src/bpm/commands/list.go

```diff
@@ -36,18 +39,62 @@ var listCommandCommand = &cobra.Command{
 func listContainers(cmd *cobra.Command, _ []string) error {
 	cmd.SilenceUsage = true
 
+	processes := []*models.Process{}
+	for _, job := range bosh.JobNames() {
+		bpmCfg := config.NewBPMConfig(bosh.Root(), job, "")
+		jobCfg, err := config.ParseJobConfig(bpmCfg.JobConfig())
+		if os.IsNotExist(err) {
+			continue
+		}
+
+		if err != nil {
+			fmt.Fprintf(cmd.OutOrStderr(), "invalid config for %s: %s", job, err.Error())
+			continue
+		}
+
+		for _, process := range jobCfg.Processes {
+			procCfg := config.NewBPMConfig(bosh.Root(), job, process.Name)
```

Comment:
Then down here we can grab the process configuration from the job configuration above.